---
title: CV
---

## Education

- 2008-2012: B.S. Mathematics (College of Creative Studies), U.C. Santa Barbara
- 2013-2020: Ph.D. Bioinformatics & Systems Biology, U.C. San Diego

## Expertise

### Software / computing

- High expertise in Linux and MacOS command line environments. Also capable of
working with Windows. I have acted as a primary system administrator for
my lab's computing resources over the past three years. See for example
my [collection of tools](https://github.com/anthony-aylward/memory_policy) for
managing memory resources on a CentOS system with several users, which
relies on my python packages [mutil](https://pypi.org/project/mutil/) and
[fancyprint](https://pypi.org/project/fancyprint/).

- Moderate expertise in programing with Python for web applications. This
includes SQL. See my personal extension to the popular
web framework [Flask](https://flask.palletsprojects.com/en/1.1.x/), comprising
[ftscursor](https://pypi.org/project/ftscursor/) and
[flask-ftscursor](https://pypi.org/project/Flask-FTSCursor/). See also my
websites: [UCSD BISB Unofficial](https://ucsd-bisb.info/auth/demo)
([GitHub](https://github.com/anthony-aylward/ucsd-bisb-unofficial)) and
[OK-scope](http://okscope.info)
([GitHub](https://github.com/anthony-aylward/okscope)). I have also made
occasional small contributions to the design and implementation of the
[Diabetes Epigenmoe Atlas](https://diabetesepigenome.org).

### Bioinformatics / statistics

- High expertise with software tools for next-generation high-throughput DNA
sequencing data, including:
  * sequence alignment (e.g.
    [BWA](http://bio-bwa.sourceforge.net),
    [Bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml), etc.)
  * variant calling (e.g. [GATK](https://software.broadinstitute.org/gatk/))
  * post-alignment analysis (e.g.
    [picard](https://broadinstitute.github.io/picard/),
    [SAMtools](http://samtools.sourceforge.net),
    [bedtools](https://bedtools.readthedocs.io/en/latest/), etc.)

- High expertise in programming with Python for bioinformatics applications.
See my various packages on [GitHub](https://github.com/anthony-aylward),
including:
  * [chipseqpeaks](https://pypi.org/project/chipseqpeaks/) : Determine binding sites of DNA-bound proteins from ChIP-seq data
  * [luciferase](https://pypi.org/project/luciferase/) : Analyze and plot results from luciferase reporter assays
  * [tssenrich](https://pypi.org/project/tssenrich/) : Calculate quality scores for accessible chromatin datasets (ATAC-seq)
  * [pydbsnp](https://pypi.org/project/pydbsnp/) : Python tools for working with dbSNP data
  * [pyhg19](https://pypi.org/project/pydbsnp/) : Python tools for working with human genome build hg19 / GRCh37
  * [py1kgp](https://pypi.org/project/py1kgp/) : Python tools for working with 1000 Genomes data
  * [seqalign](https://pypi.org/project/seqalign/) : Align NGS data using BWA or Bowtie2
  * [coloc](https://pypi.org/project/coloc/) : Python implementation of COLOC from [Giambartolomei et al.](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004383)
  * [betabernsum](https://pypi.org/project/betabernsum/) : Sums of beta-bernoulli random variables

- High expertise in programming with R and [Bioconductor](https://www.bioconductor.org) for bioinformatics and statistics. See my packages:
  * [exploreatacseq](https://github.com/anthony-aylward/exploreatacseq/blob/master/README.md) : Exploratory analysis of multiple ATAC-seq datasets
  * [refflat](https://github.com/anthony-aylward/refflat) : plot the locations of genes within a small genomic interval (this one has a good example of a [vignette](https://github.com/anthony-aylward/refflat/blob/master/vignettes/example_plot.md))
  * [betabernsum](https://github.com/anthony-aylward/betabernsum) : Sums of beta-bernoulli random variables
  * [selex](https://github.com/anthony-aylward/selex) : Utilities for statistical inference on SELEX sequencing data (powered by [multinomial regression](https://en.wikipedia.org/wiki/Multinomial_logistic_regression))
  * [npbin](https://github.com/anthony-aylward/npbin) : implementation of the NPBin test for allelic imbalance in ChIP-seq data (originally by [Zhang and Kele≈ü](https://academic.oup.com/biostatistics/article/19/4/546/4591647))

- High expertise in statistical reasoning, drawing on 6 years of classroom
training in math/statistics (including undergraduate and graduate courses) and
further independent learning during graduate school.

### Biology

- Deep domain knowledge of statistical genetics, functional genomics, and
molecular biology topics, especially as they relate to Type 1 and Type 2
diabetes (see Publications). I routinely generate data-driven hypotheses for
biological experiments such as ChIP-seq, ATAC-seq, RNA-seq, EMSA, or reporter
assays, and interpret the experimental results.

- Experience with popular biological databases such as
[dbSNP](https://www.ncbi.nlm.nih.gov/snp/),
[1KGP](https://www.internationalgenome.org),
[ENCODE](https://www.encodeproject.org),
[Roadmap](http://www.roadmapepigenomics.org),
[GTEx](http://www.roadmapepigenomics.org), and
[GWAS catalog](https://www.ebi.ac.uk/gwas/), 
including working programatically with API's.

- Held position of dkNET Fellow during a collaborative project between my lab
and [dkNET](https://dknet.org). I used the beta version of the
[Signaling Pathways Project](https://beta.signalingpathways.org/about/index.jsf)
to develop hypotheses for experiments concerning the function of genetic
variants at diabetes risk loci, such as [JAZF1](https://github.com/anthony-aylward/anthony-aylward.github.io/blob/master/jazf1-regulation.md),
for example.

### Mentorship

- I have mentored/supervised undergraduate and/or medical school students
on both computational and experimental biology research projects. See for
example Andres Padilla & Griselda Ruiz's [poster](tcf7l2-nkx63-poster.pdf) for
the [ENLACE](http://graeve.ucsd.edu/ENLACE/) program at UCSD, or Yleia
Sanchez's [ENCODE project API tool](https://github.com/yleiaa/ENCODE).

## Publications

### First Author

[Shared genetic risk contributes to type 1 and type 2 diabetes etiology](https://academic.oup.com/hmg/advance-article-abstract/doi/10.1093/hmg/ddy314/5164287) (2018)

### Other

[Pancreatic islet chromatin accessibility and conformation reveals distal enhancer networks of type 2 diabetes risk](https://www.nature.com/articles/s41467-019-09975-4) (2019)

[Gene-Wide Identification of Episodic Selection](https://academic.oup.com/mbe/article/32/5/1365/1134918) (2015)

### In progress

[A map of inflammatory-stimulated chromatin in human pancreatic islets reveals novel mechanisms of type 1 and 2 diabetes risk variants](https://github.com/anthony-aylward/islet-cytokines-outline/blob/master/abstract.md) (Accepted for poster at 2019 American Society of Human Genetics conference)
